enzy_htp/_interface/amber_interface.py

```diff
@@ -831,67 +771,11 @@ def add_charges(self, stru: struct.Structure, prmtop: str) -> None:
         for c, an, rn, aa in zip(charges, anames, res_names, stru.atoms):
 
             if not (an == aa.name and aa.residue.name == rn):
-                _LOGGER.error(
-                    f"There is a mismatch in the atoms. Make sure the supplied prmtop file is correct. Exiting.."
-                )
+                _LOGGER.error(f"There is a mismatch in the atoms. Make sure the supplied prmtop file is correct. Exiting..")
                 exit(1)
             aa.charge = c
 
         if not stru.has_charges():
-            _LOGGER.error(
-                "Not all atoms in the supplied structure were assigned charges. Exiting..."
-            )
+            _LOGGER.error("Not all atoms in the supplied structure were assigned charges. Exiting...")
             exit(1)
             pass
-
```

Comment:
Please also bring this function back. And in the best case, don't create a git records like deleting and adding the same thing again.

(I wonder why git works like this here. Shouldn't those new commits that on develop_refactor but not on your branch also applied to the final version?)

Reply:
Done with commit.